src/lib.rs

```diff
@@ -185,11 +185,6 @@ impl RustSemsimian {
         minimum_jaccard_threshold: &Option<f64>,
         minimum_resnik_threshold: &Option<f64>,
     ) -> SimilarityMap {
-        let pb = generate_progress_bar_of_length_and_message(
```

Comment:
Instead of deleting, should we just comment out this code? That way if we need it while debugging, it'll be helpful!